File: Semester 5 Fall 2024/CL3001-Computer Networks - Lab/Lab Manuals/Lab Manual # 1/C code/Q1PartC.c
#include <stdio.h>

int main(int argc,char * argv[])
{
	if(argc!=2)
	{
		printf("Invalid Parameters! Give one file name.\n");
		return 0;
	}

	FILE *read = fopen(argv[1], "r");
	if(read==NULL)
	{
		printf("Cannot open input file!\n");
		return 0;
	}

	FILE *write = fopen("PartC_Output.txt", "w"); 
	if(write==NULL)
	{
		printf("Cannot open output file!\n");
		return 0;
	}
	
	char buff[200];
	int data = fscanf(read, "%s", buff);

	while(data!=EOF)
	{
		int i = 0;
		int alphabetsPresent = 0;

		for(i = 0; buff[i] != '\0'; i++)
		{
			if ((buff[i] >= 'A' && buff[i] <= 'Z') || (buff[i] >= 'a' && buff[i] <= 'z'))
				alphabetsPresent = 1;
		}
		
		if (alphabetsPresent==0)
		{
			fputs(buff, write);
			fputc(' ', write);
		}
		data = fscanf(read, "%s", buff);
	}	

	printf("Non-alphabet word file generated successfully! with the name: 'PartC_Output.txt'\n");

	fclose(read);
	fclose(write);
	return 0;
}

File: Semester 5 Fall 2024/CL3001-Computer Networks - Lab/Lab Manuals/Lab Manual # 1/Python code/Q1PartC.py
def isNonAlphabetWord(word):
    for ch in word:
        if((ch>='a' and ch<='z') or (ch>='A' and ch<='Z')):
            return False
    return True

def main():
    outputFile= open("PartC_Output.txt",'w')
    
    with open("Input File.txt",'r') as f:
        for line in f:
            words=line.split()
            for word in words:
                if(isNonAlphabetWord(word)==True):
                    outputFile.write(word)
                    outputFile.write(" ")
            outputFile.write("\n")
    
            
    outputFile.close()




main()